AsyncDisplayKit/ASCollectionView.mm

```diff
@@ -301,10 +302,13 @@ - (void)setAsyncDataSource:(id<ASCollectionViewDataSource>)asyncDataSource
     _asyncDataSource = nil;
     _proxyDataSource = _isDeallocating ? nil : [[ASCollectionViewProxy alloc] initWithTarget:nil interceptor:self];
     _asyncDataSourceImplementsConstrainedSizeForNode = NO;
+    _asyncDataSourceImplementsNodeBlockForItemAtIndexPath = NO;
   } else {
     _asyncDataSource = asyncDataSource;
     _proxyDataSource = [[ASCollectionViewProxy alloc] initWithTarget:_asyncDataSource interceptor:self];
     _asyncDataSourceImplementsConstrainedSizeForNode = ([_asyncDataSource respondsToSelector:@selector(collectionView:constrainedSizeForNodeAtIndexPath:)] ? 1 : 0);
+
```

Comment:
Newline not needed, or add one before the two respondsToSelector calls.  Could also remove the odd / ridiculous ? 1 : 0 on the previous line ha